src/forms/index.tsx

```diff
@@ -216,28 +252,64 @@ export function Select(props: RenderableProps<ISelectProps>) {
 
 export interface ICheckboxProps {
   value?: boolean;
+  checked?: boolean;
   disabled?: boolean;
   onChanged?: (ev: Event) => void;
+  onChange?: (ev: Event) => void;
   id?: string;
   name?: string;
+  onFocus?: (ev: Event) => void;
+  onBlur?: (ev: Event) => void;
 }
 
 export function Checkbox(props: RenderableProps<ICheckboxProps>) {
+  if (IS_DEVELOPMENT && props.value) {
+    console.warn("Checkbox.value will be deprecated in a future release. Please use Checkbox.checked instead.");
+  }
+  if (IS_DEVELOPMENT && props.onChanged) {
+    console.warn("Checkbox.onChanged will be deprecated in a future release. Please use Checkbox.onChange instead.");
+  }
+
+  const changeEvent = props.onChanged || props.onChange;
+  const checked = props.value || props.checked;
```

Comment:
@SolarLiner thoughts on this approach for supporting both API's? I wasn't sure which to prioritize. A coin flip made the to-be-deprecated ones take priority :lad

Reply:
Actually, in my mind it would make sense that the new value takes precedence. In any case, both don't break existing code, however if one defines both, we want the new behavior prioritized over the old, therefore `onChange` and `checked` should be checked first.

Reply:
Sounds good to me. I'll push these changes soon.